### DISCUSSION TITLE
Hosting budibase

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#7457] [Help] [answered]
Hello, 

After sometime developing the apps in budibase using docker i was thinking about testing it in a simple home network hosted using a Raspberry pi 3B+ but i've read that budibase isn't supported due to raspberry pi processor architecture, is this true?
If so, what are the alternatives? 
I really wanted to host it in a static point (raspberry or pc) and then access the apps in any devices connected to the network by browser.

Also, how can i transfer the whole budibase image? (Users, Apps).

Many thanks!

**Comment by melohagan** [comment, marked as answer]
From a quick look at Wikipedia, it appears Raspberry Pis are ARM compatible. This means that it may be possible to run it via emulation: https://docs.budibase.com/docs/docker-compose#setting-up-budibase

With that said, emulation will require more computing resources, which the PI may not be able to handle.

**Reply by mike12345567** [reply]
Hi @insomniacj93 it should also be noted that our Budibase single image (found [here](https://docs.budibase.com/docs/docker)) is natively ARM compatible - if you use this rather than our Kubernetes or Docker-compose installation methods you will not require emulation.

**Reply by insomniacj93** [reply]
@melohagan @mike12345567 Thank you guys. I'll give it a try!
